.gitignore

```diff
@@ -21,3 +21,7 @@ build
 # matlab / octave
 *.mat
 octave-workspace
+
+# clion
+.idea/
+cmake-build-*/
```

Comment:
quite a few lab members use clion so that addition to gitignore is fine. We currently don't have a universal `.gitignore`, I personally don't see the problem adding custom gitignore rules as they come (since they are pretty rare).